### DISCUSSION TITLE
ViewInspector does not support watchos...don't know why....

### DISCUSSION THREAD

**Discussion by ghost** [nalexn/ViewInspector#282] [Q&A]
Xcode version: 15.0.1
Cocoapods version: 1.12.1
watch extension minimum deployment target: 7.0

Hello, I am trying to import ViewInspector using cocoapods to my project. 
however, I keep getting this error:

<img width="646" alt="스크린샷 2023-11-29 오후 2 01 32" src="https://github.com/nalexn/ViewInspector/assets/97498615/686fbef7-6c4b-462a-ade4-db8cfd718d76">

I did try to import the library to test target.
```
target 'watch Extension' do
    platform :watchos, '7.0'
    pod "Communicator"
    pod 'AppleWatchLogger'
    pod 'KeychainAccess'

    target 'watch ExtensionTests' do
        inherit! :search_paths
        pod 'ViewInspector'
    end
end
```

However, I still get the same error. Please help...

**Comment by nalexn** [comment]
Hey, watchOS is supported, however, its support isn't declared explicitly in the podspec. This is due to limitations of pod deployment flow, where it attempts to run the main test suit before the deploy and fails (because the project has a separate watchOS project for tests).
A workaround is to add the library with the SPM or carthage.

**Reply by ghost** [reply]
Thank you for the answer. I will try to add the library using SPM :)
